kubernetes/base/config/__init__.py

```diff
@@ -33,8 +33,10 @@ def load_config(**kwargs):
     can be passed to either load_kube_config or
     load_incluster_config functions.
     """
-    if "kube_config_path" in kwargs.keys() or exists(expanduser(KUBE_CONFIG_DEFAULT_LOCATION)):
+    if "config_file" in kwargs or exists(expanduser(KUBE_CONFIG_DEFAULT_LOCATION)):
         load_kube_config(**kwargs)
+    elif "config_dict" in kwargs:
+        load_kube_config_from_dict(**kwargs)
     else:
         print(
             "kube_config_path not provided and "
```

Comment:
could you fix the above line: kube_config_path

Reply:
which line?